lib/js/components/Outline/OutlineItem/OutlineItem.vue

```diff
@@ -26,8 +26,15 @@
 				:icon="iconLeft"
 				:size="ICON_SIZES.X_SMALL"
 			/>
-			<span class="ds-outlineItem__text">
+			<span v-else class="ds-outlineItem__text">
 				<span class="ds-outlineItem__label" :class="{ '-ds-uppercase': isLabelUppercase }">
+					<template v-if="$slots.label">
+						<slot name="label" />
```

Comment:
should we mark it in a storybook that this component accept both prop and slot?

Reply:
yeah, u right 😒